homeassistant/components/daikin/const.py

```diff
@@ -21,6 +26,21 @@
         CONF_ICON: "mdi:thermometer",
         CONF_TYPE: SENSOR_TYPE_TEMPERATURE,
     },
+    ATTR_TOTAL_POWER: {
+        CONF_NAME: "Total Power Consumption",
+        CONF_ICON: "mdi:flash",
```

Comment:
Do not set a custom icon for entities with device class. They already have an icon set in the frontend.

Reply:
As all the sensors (even climate ones) now have a class, I've simply removed the `icon` property